website/content/api-docs/operator/raft.mdx

```diff
@@ -145,3 +145,41 @@ $ curl \
     --request DELETE \
     "http://127.0.0.1:8500/v1/operator/raft/peer?address=1.2.3.4:5678"
 ```
+
+## Transfer Raft Leadership
+
+This endpoint transfer the raft leadership from the current raft leader to a new node or
+the node with the provided id.
+
+
+If ACLs are enabled, the client will need to supply an ACL Token with `operator`
+write privileges.
+
+| Method   | Path                  | Produces           |
+| -------- | --------------------- | ------------------ |
+| `POST` | `/operator/raft/transfer-leader` | `application/json` |
+
+The table below shows this endpoint's support for
+[blocking queries](/api-docs/features/blocking),
+[consistency modes](/api-docs/features/consistency),
+[agent caching](/api-docs/features/caching), and
+[required ACLs](/api-docs#authentication).
+
+| Blocking Queries | Consistency Modes | Agent Caching | ACL Required     |
+| ---------------- | ----------------- | ------------- | ---------------- |
+| `NO`             | `none`            | `none`        | `operator:write` |
+
+The corresponding CLI command is [`consul operator raft tranfer-leader`](/commands/operator/raft#tranfer-leader).
+
+### Query Parameters
+
+- `id` `(string: "")` - Specifies the ID of the raft peer to transfer leadership to, if empty the leadership
+will be transferred to a random server.
```

Comment:
Is the "ID" here a "node ID"? Or some either kind of ID? We should make that explicit.

```suggestion
- `id` `(string: "")` - Specifies the node ID of the raft peer to transfer leadership to.
If empty, leadership will be transferred to a random server agent.
```